packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -469,6 +469,26 @@ export type CustomerDescriptions = {
     customerId: string;
 };
 
+export type Product = {
```

Comment:
The naming of these types is to generic. They should reflect that they are MetaData parameters. Something like MetaDataDescriptionParams? I also propose it can be one type. Maybe discuss Thursday?

```suggestion
export type MetadataDescriptionParams = {
    id: string:
    desription: id;  
}
```